### ISSUE TITLE
Too small text areas

### ISSUE THREAD

**Issue by HencoDesu** [Wolfteam/GenshinDb#29]
In several places name of the item doesn't fit and ended with ellipsis. See screenshot below
![изображение](https://user-images.githubusercontent.com/17922521/110355296-20228300-804a-11eb-8ee3-bdfc46d69fb2.png)
I think that area for names can have a second line if it too long

**Comment by kasproTV** [comment]
I propose to make the image of the material clickable to get its full name.

**Comment by Wolfteam** [comment]
Yeah I Know, can't remember why I ended up implementing it like that (I mean, the technical reason).
Btw you can long click the text to get the full name xd
